### ISSUE TITLE
list index out of range in vad.py

### ISSUE THREAD

**Issue by ovhpa** [guillaumekln/faster-whisper#118]
Hello and thank you for your nice whisper implementation!

When processing Audio chunks (for a real time application), I encounter the following error:
```
  File "vad.py", line 206, in get_original_time
    total_silence_before = self.total_silence_before[chunk_index]
IndexError: list index out of range
```

I'm not sure what is the cause. 
A quick workaround is to insert an index test:
```
206,208d205
<         if chunk_index >= len(self.total_silence_before):
<             chunk_index = -1
< 
214c211,214
<         return bisect.bisect(self.chunk_end_sample, sample)
---
>         return min(
>             bisect.bisect(self.chunk_end_sample, sample),
>             len(self.chunk_end_sample) - 1,
>         )
```

I didn't PR because you might want to investigate the cause.

Sincerely,

**Comment by guillaumekln** [comment]
Hi,

Are you using the latest version? This error was supposedly fixed in #107 which was released in version 0.4.1.

**Comment by ovhpa** [comment]
Ah, I could swear I did the update, but I am actually running version 0.4.0 ^^'

I'm testing 0.4.1 now, it looks OK.
Sorry for that m(_ _)m
